apps/web/public/Icons/_contact.html

```
@@ -0,0 +1,8 @@
+<body style="margin:0;background:#cdd6e0;display:flex;flex-wrap:wrap;gap:14px;padding:16px;font:12px sans-serif">
+<div style="background:#fff;padding:10px;border-radius:8px;text-align:center"><img src="MainLogo.png" style="height:40px"><div>MainLogo.png (on white)</div></div>
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)